benchmark/run.py

```diff
@@ -1,10 +1,9 @@
 import timeit
 from pathlib import Path
 
-_parsers = []
+_parsers = [('pytomlpp', pytomlpp.loads)]

```

Comment:
I didn't understand why you did the append when you can initialise it with those values if that append is always called. Was there any reason to do this?